crates/admin/Cargo.toml

```diff
@@ -22,7 +21,7 @@ metadata-api = []
 workspace-hack = { version = "0.1", path = "../../workspace-hack" }
 
 restate-admin-rest-model = { workspace = true, features = ["schema"] }
-restate-bifrost = { workspace = true }
+restate-bifrost = { workspace = true, features = ["local-loglet"] }
```

Comment:
imo its still an issue if admin relies on local loglet as it means you cant import the cluster ctrl proto without importing rocksdb. that said i can work around it by copying the proto generated file over to my repo

Reply:
Maybe the proto file should just be moved to `restate-types`.